Improving readme

helrond opened the pull request:
Adding some more information to README, including an infrastructure diagram and related repositories.

helrond committed:
improving readme

HaSistrunk committed:
fix image

HaSistrunk committed:
language tweaks

HaSistrunk commented on README.md:
```diff
@@ -1,20 +1,29 @@
+## Related Repositories
+
+- [digital_ingest_assembly](https://github.com/RockefellerArchiveCenter/digital_ingest_assembly/) - creates Archivematica-compliant Submission Information Packages (SIPs).
+- [digital_ingest_discovery](https://github.com/RockefellerArchiveCenter/digital_ingest_discovery/) - discovers 
+and prepares the SIPs for processing.
```
Are these SIPs at this point? Or do they become SIPs after the assembly?

helrond replied:
You're right, they're technically only SIPs once they get started in Archivematica.

HaSistrunk commented:
This is really useful! I'm trying to match the diagram up with the repositories and their actions.
- "ingest bucket" is an S3 bucket, right?
- "Trigger lambda" (and "notifications lambda"?) are related to the digital_ingest_trigger?
- Since this is the spot where all the parts of the pipeline are referenced, it would be useful to add something about the Zodiac Backend API and Zodiac frontend.

helrond committed:
adds info about zodiac frontend and backend, reorders repositories by process.

helrond committed:
add info about which type of thing these are;

helrond committed:
update diagram

helrond commented:
Thank you! I made a couple of changes:
- reordered the list of related repos so it follows the flow of the overall process
- added the zodiac repos to the list
- added some detail about how each of these services are implemented (ECS Task, Lambda, etc)
- updated the diagram to show the zodiac apps
